resources/views/layout-extern/index.blade.php

```diff
@@ -60,14 +102,8 @@
 			</div>
 		</section>
 
-		<!--
-		<section id="two" class="wrapper-img">
-			<img src="/images/vereniging.jpg"/>
-		</section>
-		-->
-
-		<!-- Three -->
-		<section id="three" class="wrapper kleur3">
+		<!-- Four -->
+		<section id="four" class="wrapper alt kleur4">
```

Comment:
Deze comment was er voor als we ooit nog paginabrede plaetjes willen (wat in het promocie ontwerp stond)